pages/data-providers/[data-provider-id]/overview.css

```diff
@@ -19,3 +19,13 @@
     grid-template-columns: repeat(3, 1fr);
   }
 }
+
+.overview-card {
+  display: flex;
+  flex-direction: column;
```

Comment:
You should combine this with `justify-content: flex-start`

Reply:
I wanted to keep all other elements with width 100%.

Reply:
It's align-items stretch in this case. This is default too.